plugins/cost-insights/api-report.md

```diff
@@ -586,8 +586,8 @@ export interface ResourceData {
 // @public (undocumented)
 export type TooltipItem = {
   fill: string;
-  label: string;
-  value: string;
+  label: NameType | undefined;
+  value: ValueType;
```

Comment:
This seems like a widening of the possibilities, is that right? Could we have kept this as string + string only, and instead adapted the code underneath? I'm just thinking that maybe, widening the possibilities here could be an entirely different pull request if there turned out to be specifically a need for it.

Reply:
It makes sense, I'll undo these changes and keep them as a string